src/Sarif.Multitool.Library/SuppressOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("suppress", HelpText = "Enrich a SARIF file with additional data.")]
+    public class SuppressOptions : SingleFileOptionsBase
+    {
+        [Option(
+            "justification",
+            HelpText = "A string that provides the rationale for the suppressions",
+            Required = true)]
+        public string Justification { get; set; }
+
+        [Option(
+            "alias",
+            HelpText = "The account name associated with the suppression.")]
+        public string Alias { get; set; }
```

Comment:
>Alias [](http://example.com/codeflow?start=22&length=5)

just a thought, outside of Microsoft user may not immediately recognize Alias as Alias for user. do we like to change it to more common like user name, user alias, account name